### PR TITLE
Ticket/master/docs

### PR DESCRIPTION
**Pull request by kbarber**
A series of small improvements to initial setup docs.

### CODE REVIEW

**Comment by kbarber** [comment]
@jeffmccune I didn't see a card created for this pull request in your trello project for puppet-webhooks, I created card 33 as a placeholder anyway.

**Comment by jeffmccune** [comment]
> @jeffmccune I didn't see a card created for this pull request in your trello project for puppet-webhooks, I created card 33 as a placeholder anyway.

@kbarber Yeah, this brings up an interesting topic.  @daniel-pittman mentioned that it's nice for things like this to make some reasonable guarantees around how many times the service will retry delivery.  I wonder if this particular card got dropped on the floor because of an error talking to Trello or something.

In any event, it would be nice to add some delivery retry functionality to this app.

File: README.md [hunk 1]
```diff
@@ -22,10 +22,15 @@ This project performs a job or jobs when a pull request event occurs on
 Quick Start
 ====
 
-First, review the [Heroku Quickstart][quickstart] guide.  Next, we simply need
-to create a copy of this application.  run `heroku create` which will add a git
-remote named `heroku` to the git repository.  Deployments will happen with a
-simple `git push heroku`.
+First, review the [Heroku Quickstart][quickstart] guide. Create an account and
+install the Toolbelt. To use this code you'll need to verify your account by
```
Comment (jeffmccune) [inline README.md:26]: Pretty minor, but perhaps "to use this code on Heroku you'll need."  I actually got pretty darn far using the foreman application from the tool belt.

Comment (jeffmccune) [inline README.md:26]: I added a commit that mentions `RACK_ENV=development foreman start`.

**Pull request merged**